client/resources/pixel-buffer.cpp

```diff
@@ -86,6 +86,12 @@ namespace tec {
 					if (b < 3) std::fputc(p[i], stderr); // only output RGB
 				}
 				break;
+			case ImageColorMode::MONOCHROME:
+				break;
```

Comment:
Most of these breaks can be removed as the cases will falldown till exit regardless.